src/main/java/edu/unc/lib/boxc/migration/cdm/services/FieldAssessmentTemplateService.java

```diff
@@ -0,0 +1,141 @@
+                .collect(Collectors.toList());
+
+        // Create workbook and sheet
+        XSSFWorkbook workbook = new XSSFWorkbook();
+        Sheet sheet = workbook.createSheet("Template");
+
+        // Style spreadsheet
+        XSSFFont defaultFont = workbook.createFont();
+        defaultFont.setFontHeightInPoints((short) 11);
+        defaultFont.setFontName("Calibri");
+        defaultFont.setColor(IndexedColors.BLACK.getIndex());
+        defaultFont.setBold(false);
+        defaultFont.setItalic(false);
+
+        XSSFFont fontBold = workbook.createFont();
+        fontBold.setFontHeightInPoints((short) 11);
+        fontBold.setFontName("Calibri");
+        fontBold.setColor(IndexedColors.WHITE.getIndex());
+        fontBold.setBold(true);
+        fontBold.setItalic(false);
+
+        XSSFCellStyle defaultStyle = workbook.createCellStyle();
+        defaultStyle.setFont(defaultFont);
+
+        XSSFCellStyle headerStyle = workbook.createCellStyle();
+        headerStyle.setFillForegroundColor(new XSSFColor(new java.awt.Color(0, 102, 204), null));
+        headerStyle.setFillPattern(FillPatternType.SOLID_FOREGROUND);
+        headerStyle.setFont(fontBold);
+
+        SheetConditionalFormatting sheetCF = sheet.getSheetConditionalFormatting();
+        ConditionalFormattingRule rule1 = sheetCF.createConditionalFormattingRule("=MOD(ROW(),2)=0");
+        PatternFormatting fill = rule1.createPatternFormatting();
+        XSSFColor color = new XSSFColor(new java.awt.Color(0, 102, 204), null);
+        color.setTint(0.85);
+        fill.setFillBackgroundColor(color);
+        fill.setFillPattern(PatternFormatting.SOLID_FOREGROUND);
+        CellRangeAddress[] regions = {CellRangeAddress.valueOf("A1:K200")};
+        sheetCF.addConditionalFormatting(regions, rule1);
+
+        // Add a header row, and freeze it
+        Row header = sheet.createRow(0);
+        sheet.setAutoFilter(new CellRangeAddress(0, 0, 0, 10));
+        sheet.createFreezePane(0,1);
+        header.setRowStyle(headerStyle);
+        header.createCell(0).setCellValue("Field Label");
+        header.createCell(1).setCellValue("Alias");
+        header.createCell(2).setCellValue("MD Source");
+        header.createCell(3).setCellValue("All fields blank?");
+        header.createCell(4).setCellValue("Some fields blank?");
+        header.createCell(5).setCellValue("Repeated fields?");
+        header.createCell(6).setCellValue("Retain?");
+        header.createCell(7).setCellValue("Display?");
+        header.createCell(8).setCellValue("MODS mapping");
+        header.createCell(9).setCellValue("Notes");
+        header.createCell(10).setCellValue("Remediation needed");
+
+        // Add a row for every field in field info object and fill in default values for all blank ? fields
+        int rowNum = 0;
+        for (int i = 0; i < fields.size(); i++) {
+            rowNum++;
+            Row row = sheet.createRow(rowNum);
```

Comment:
since this is the only usage of rowNum, you could safely remove the variable and change this to `sheet.createRow(i + 1)`